draft-irtf-cfrg-opaque.md

```diff
@@ -598,11 +586,11 @@ Steps:
 1. (kU, _) = KeyGen()
 2. Z = Evaluate(kU, request.data)
 3. Create RegistrationResponse response with
-     (Z, pkS, secret_credentials_list, cleartext_credentials_list)
+     (Z, pkS)
 4. Output (response, kU)
 ~~~
 
-#### FinalizeRequest
+#### FinalizeRequest {#finalize-request}
 
 ~~~
 FinalizeRequest(pwdU, skU, blind, request, response)
```

Comment:
Also unrelated, but we can remove `request` here, as it's not used.

Reply:
It is used in line 468:

```
The full procedure for constructing `Envelope` and `InnerEnvelope` from
`SecretCredentials` and `CleartextCredentials` is described in {{finalize-request}}.
```

Reply:
I'll address this in a separate PR, since it seems related to the module boundary we discussed.